blueprint/client-devices.md

```diff
@@ -1151,7 +1151,16 @@ If there is a requirement to utilise the local Administrator account in an on-pr
 
 LAPS allows system administrators the ability to set a different, random password for the common local administrator account on each computer in the domain and store the password for the computer's local administrator account in Active Directory, secured in a confidential attribute in the computer's corresponding Active Directory object.
 
-Local Administrator Design Decisions for all agencies and implementation types.
+For Administrative staff to manage and support endpoints the adding of groups can be configured to the local Administrators group, granting administrator privileges to the endpoint. This is through the adding of Azure Active Directory groups via a custom Intune configuration profile using LocalUsersAndGroups policies. Dependencies for this feature are detailed below:
+
+* Windows 10 Build 20H2 or higher
```

Comment:
Does this need to match up with https://desktop.gov.au/blueprint/client-devices.html#operating-system

`Windows 10 Build: 1909`

I think 1909 is mentioned in various documents.

Reply:
For this feature to work Version 20H2 or higher must be used as per MS documentation https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-localusersandgroups.  If all features have to go of 1909 build this will have to be withdrawn

Reply:
@zedzedzed  this could be simplified by using the native Azure Portal place for this (if required as would lower E8 ML) (https://docs.microsoft.com/en-us/azure/active-directory/devices/assign-local-admin#manage-the-device-administrator-role) Default stance for ML3 would be to use PIM/Device Admins role when needing to elevate. Thoughts?

Reply:
@zedzedzed this PR can be closed off. E8 default stance is use PIM.